.github/workflows/docker.yml

```
@@ -0,0 +1,32 @@
+name: "docker"
+on:
+  push:
```

maybe we should push on master branch too with tag `latest`.

```suggestion
  push:
    branches:
      - master
```